### ISSUE TITLE
[spline-reco] Tests fail with recent photospline bugfix (Trac #1724)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1716]
http://code.icecube.wisc.edu/projects/icecube/changeset/146715/IceCube

That caused spline-reco tests to fail on all bots.  Attached is the output.  It looks like the result changed just enough to cause the regression tests to fail.  This needs to be verified.

Currently the tests are disabled.  These need to be fixed and then re-enabled.

Migrated from https://code.icecube.wisc.edu/ticket/1724
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:47", 
    "description": "http://code.icecube.wisc.edu/projects/icecube/changeset/146715/IceCube\n\nThat caused spline-reco tests to fail on all bots.  Attached is the output.  It looks like the result changed just enough to cause the regression tests to fail.  This needs to be verified.\n\nCurrently the tests are disabled.  These need to be fixed and then re-enabled.\n\n", 
    "reporter": "olivas", 
    "cc": "jvansanten", 
    "resolution": "fixed", 
    "_ts": "1550067167842669", 
    "component": "combo reconstruction", 
    "summary": "[spline-reco] Tests fail with recent photospline bugfix", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-06-02T20:50:23", 
    "milestone": "", 
    "owner": "mvoge", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160602T20:50:58: olivas changed attachment from "" to "spline-reco-test-output"

**Comment by icecube-trac** [comment]
Trac update at 20160729T12:09:49: 
* kjmeagher changed status from "new" to "closed"
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher commented:

> In changeset 148896:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="148896"
> update regression tests after legimate improvement to `splinetable_convolve()` fixes #1716
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:47: 
* olivas commented:

> Milestone June 2016 deleted


* olivas changed milestone from "June 2016" to ""
